src/sas/qtgui/Perspectives/Corfunc/UI/CorfuncPanel.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>762</width>
-    <height>1182</height>
+    <height>1228</height>

```

Comment:
will this not overflow the canvas at lower resolutions?
This has been a rather common issue with the corfunc widget, so we might want to start thinking about adding tabs and changing the layouts... (obviously not here!)

Reply:
yeah, there's more fields to come too!

Reply:
I will have a look at this with #2110 or something similar